packages/fx-core/tests/common/tools.test.ts

```diff
@@ -192,24 +192,5 @@ describe("tools", () => {
       chai.assert.isDefined(result);
       chai.assert.isTrue(result);
     });
-
```

Comment:
Why remove this case?

Reply:
The function `canAddSso()` has been changed and this test case is outdated and broken so I removed this case.